src/app/settings/views/Scripts/ScriptDetails/ScriptDetails.tsx

```diff
@@ -43,11 +56,29 @@ const ScriptDetails = ({ id }: Props): JSX.Element => {
   }
 
   return (
-    <Row>
-      <Col size={10}>
-        <Code className="u-no-margin--bottom">{script}</Code>
-      </Col>
-    </Row>
+    <>
+      <Row>
+        <Col size={10}>
+          <Code className="u-no-margin--bottom">{script}</Code>
+        </Col>
+      </Row>
+      {isCollapsible && (
+        <Row>
+          <Col className="u-flex--end" emptyLarge={8} size={4}>
+            <Button
+              appearance="link"
+              aria-label="Hide details"
```

Comment:
I think you should be okay to not have this here, since the button has text within it that should be picked up by screen readers.